lib/backend/k8s/syncer.go

```diff
@@ -313,8 +313,19 @@ func (syn *kubeSyncer) readFromKubernetesAPI() {
 
 	log.Info("Starting Kubernetes API read loop")
 	for {
+		needSync := false
```

Comment:
I'm guessing all of this will be blatted over by @robbrockbank;

Reply:
that's what i'm hoping.